From 1357e7047872f2b1d48563580ba3b27a2da43119 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 22 Nov 2017 15:24:06 +0800
Subject: [PATCH 2876/5242] MLK-16942-2: dts: Remove disp_apb clock rate
 setting

commit  2b8e24ada4b63d5fc430a09bc8e77f9136efc66d from
https://source.codeaurora.org/external/imx/linux-imx.git

disp_apb clock is hdmi core root.
It has enabled and configurated by ROM code and
clock rate should keep in 133MHz when device bootup.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit 5dfa629c82af77408c8f0ed8f01a5e68e80899ba)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 796a646..c4a65cb 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -550,16 +550,13 @@
 		clock-names = "apb", "axi", "rtrm", "pixel", "dtrc";
 		assigned-clocks = <&clk IMX8MQ_CLK_DC_PIXEL_SRC>,
 				  <&clk IMX8MQ_CLK_DISP_AXI_SRC>,
-				  <&clk IMX8MQ_CLK_DISP_APB_SRC>,
 				  <&clk IMX8MQ_CLK_DISP_RTRM_SRC>,
 				  <&clk IMX8MQ_CLK_DISP_RTRM_PRE_DIV>;
 		assigned-clock-parents = <&clk IMX8MQ_VIDEO_PLL1_OUT>,
 					 <&clk IMX8MQ_SYS1_PLL_800M>,
-					 <&clk IMX8MQ_SYS2_PLL_125M>,
 					 <&clk IMX8MQ_SYS1_PLL_800M>;
 		assigned-clock-rates = <594000000>,
 				       <800000000>,
-				       <125000000>,
 				       <400000000>,
 				       <400000000>;
 		status = "disabled";
-- 
1.7.9.5

